schema/deploy/ciip_functions/ciip_incentive.sql

```diff
@@ -5,7 +5,7 @@
 
 begin;
 -- Function takes application id and version number:
-create or replace function ggircs_portal.ciip_incentive(app_id int , version_no int )
+create or replace function ggircs_portal.ciip_incentive(app_id text , version_no text)
```

Comment:
That will do for now. Ultimately, this should be a computed field of `applicationRevision`

Reply:
@matthieu-foucault  I'll go ahead and make it into a computed column